ui/app/src/components/PluginCard/PluginCard.tsx

```diff
@@ -333,7 +338,14 @@ function PluginCard(props: PluginCardProps) {
           />
         </CardActionArea>
       )}
-      <CardActionArea onClick={() => onPluginSelected(plugin, 1)}>
+      <CardActionArea
```

Comment:
When the card itself it's disabled, make sure the cursor is default (i.e. not the hand, which means "interactive")